app.js

```diff
@@ -1,13 +1,15 @@
 const express = require("express");
 const app = express();
+app.use(express.json())
 const endpoints = require("./endpoints.json");
-// const { api , getApiTopics } = require("./controller")
+const { api, getApiTopics } = require("./controller");
 
-app.get("/api", (req, res) => {
-  console.log("inside app")
+app.get("api", (req, res) => {
   res.status(200).send({ endpoints });
 });
 
+app.get("/api", api)
```

Comment:
It's good convention to name functions as verbs to show what they do - e.g. instead of api, use getApi